back/index.js

```diff
@@ -53,11 +53,7 @@ app.route('/register').post(async (request, response) => {
 									.status(500)
 									.send("Erreur pendant l'inscription.")
 							} else {
-								jwt.sign(user.email, secret, (err, token) => {
-									response.json({
-										token: token
-									})
-								})
+								response.redirect('http://localhost:3001/login')
```

Comment:
this should not be done by an API: the API sends a status and the front decides what to do from it (redirection or not) + this disables the login process ? no more token sent back ?

Reply:
I don't know what happened I'll delete this branch and redo my work of this morning on a new one